src/Hedgehog/Gen.fs

```diff
@@ -354,7 +368,7 @@ module Gen =
 
     /// Generates a random string.
     let string : Gen<string> =
-        choice [alphaNum; charBounded] |> string'
+        choice [alphaNum; unicodeAll] |> string'
```

Comment:
*That* was exactly what I was thinking right now... We're going to have `Gen.fs` in line with the Haskell one anyway, so good idea. I'll undo 4979fb5 also.